src/ugeneui/src/app_settings/user_apps_settings/UserApplicationsSettingsGUIController.h

```diff
@@ -82,6 +82,10 @@ class UserApplicationsSettingsPageWidget : public AppSettingsGUIPageWidget, publ
 
 private slots:
     void sl_transFileClicked();
+
+private:
+    //Keys
```

Comment:
Please add a detailed documentation why this field exists and how it is used.

Reply:
I started, but forgot to finish) Done